src/testUtils/mocks/apollo.ts

```diff
@@ -0,0 +1,32 @@
+import { InMemoryCache } from 'apollo-cache-inmemory';
```

Comment:
Logic related to the mock apollo client can't be split between this file and the polkadot one. I suggest renaming testUtils/mocks/polkadot.ts to testUtils/mocks/dataSources.ts and moving this logic over there. For the constructor, use the same logic that is being used in that file (see keyringConstructorStub)

Reply:
(While you're at it, rename all references to polkadotMockUtils to dsMockUtils)